Assets/AnkrSDK/Scripts/Core/Utils/EventFilterHelper.cs

```diff
@@ -6,35 +6,18 @@ namespace AnkrSDK.Core.Utils
 {
 	public static class EventFilterHelper
 	{
-		public static NewFilterInput CreateEventFilters(EventBase eventHandler, EventFilterData evFilter = null)
+		public static NewFilterInput CreateEventFilters<TEvDto>(string contractAddress, EventFilterData evFilter = null)
```

Comment:
add in code documentation that TEvDTO should contain [Event] attribute and for parameters to work it is required that parameters are marked with [Parameter] attribute

Reply:
Up

Reply:
https://ankrnetwork.atlassian.net/browse/GD-79